mage_ai/api/resources/PipelineScheduleResource.py

```diff
@@ -216,6 +225,22 @@ def update(self, payload, **kwargs):
 
             self.tag_associations_updated = tag_associations_updated + tag_associations_to_keep
 
+        trigger_as_code_exists = self.model.trigger_as_code_exists
+        if trigger_as_code_exists:
+            pipeline_uuid = self.model.pipeline_uuid
+            trigger = Trigger(
```

Comment:
You can create the trigger after the db model is saved so that you can get the fields from the model directly.